src/ResourceManager/Dns/Commands.Dns/Models/DnsClient.cs

```diff
@@ -424,13 +438,16 @@ private static DnsRecordSet GetPowerShellRecordSet(string zoneName, string resou
             return new DnsRecordSet
             {
                 Etag = mamlRecordSet.Etag,
+                Id = mamlRecordSet.Id,
                 Name = mamlRecordSet.Name,
                 RecordType = recordType,
                 Records = GetPowerShellRecords(mamlRecordSet),
                 Metadata = TagsConversionHelper.CreateTagHashtable(mamlRecordSet.Metadata),
                 ResourceGroupName = resourceGroupName,
                 Ttl = (uint) mamlRecordSet.TTL.GetValueOrDefault(),
```

Comment:
Still casting to a `uint` here.

Reply:
updated the PR and removed the nullable.